x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/RestEnrichTestCase.java

```diff
@@ -144,7 +145,11 @@ public void wipeTestData() throws IOException {
     public void testNonExistentEnrichPolicy() throws IOException {
         ResponseException re = expectThrows(
             ResponseException.class,
-            () -> RestEsqlTestCase.runEsqlSync(new RestEsqlTestCase.RequestObjectBuilder().query("from test | enrich countris"), List.of())
+            () -> RestEsqlTestCase.runEsqlSync(
+                new RestEsqlTestCase.RequestObjectBuilder().query("from test | enrich countris"),
+                NO_WARNINGS,
```

Comment:
Can't you use here `RestEsqlTestCase.runEsqlSync(RequestObjectBuilder requestObject)`?